packages/xo-web/src/common/xo/index.js

```diff
@@ -3796,3 +3798,29 @@ const _callGithubApi = async (endpoint = '') => {
 export const getMasterCommit = () => _callGithubApi('/commits/master')
 
 export const compareCommits = (base, head) => _callGithubApi(`/compare/${base}...${head}`)
+
+export const uuidToLink = text => {
```

Comment:
This should be a component so that it can connect to the store and react to changes.